### ISSUE TITLE
Request: Manual Line Matching

### ISSUE THREAD

**Issue by Caraamon** [pnedev/comparePlus#398]
Due to quirks in the way things are matched, it's not always as reliable as I'd like. I would love to be able to add manual match points, even if it requires creating a special file to store the info. I've seen a few other comparison programs that can do this, but they always are missing something else helpful that Notepad++ or this addon does.

In a perfect world, I'd also be able to link and explicitly label it as being a moved section, but I suspect that's a bit of a dream.

**Comment by pnedev** [comment]
@Caraamon ,
I agree that manual match points are very useful tool to customize and guide the compare process.
I'll consider implementing such functionality (at least the most basic one as a start - setting two lines that are supposed to be matched - unlimited such points of course) but I don't work on ComparePlus very often due to priorities so I can't promise anything.

**Comment by Caraamon** [comment]
I have really come to really appreciate your addon recently as I've found more uses for it. There are a lot of programs that compare documents, but they all seem to lack one or two things I really want, and even without manual points, it's still the best option I've tried so far.

I'd of course prefer a speedier update, but it's a free product, so I'd have to be a total ass to complain about it not being higher priority.
